src/main/java/com/github/erizo/gradle/JcstressPlugin.java

```diff
@@ -222,7 +222,7 @@ private JcstressTask addJcstressTask() {
         final JcstressTask jcstressTask = project.getTasks().create(TASK_JCSTRESS_NAME, JcstressTask.class);
 
         jcstressTask.dependsOn(jcstressJarTask);
-        jcstressTask.setMain("org.openjdk.jcstress.Main");
```

Comment:
Good evening Carl.

Thanks for the Pull Request! Let me check this within a few days. I'll need to verify which lowest Gradle version has this method. 

Thank you again!

Reply:
FWIW I tried to see when the mainClass attribute was added, but it's not well documented. Sorry for the chore!

Reply:
Hello Carl.

Now I see that the tests were disabled.

I fixed the problem i na bit different way - for now it'll work. Could you review these changes?

https://github.com/reyerizo/jcstress-gradle-plugin/pull/48

Thanks!